source/scpd/types/XDRBase.d

```diff
@@ -45,48 +45,32 @@ uint size32(size_t s)
 ////////////////////////////////////////////////////////////////
 
 //! Generic class of XDR unmarshaling errors.
-class xdr_runtime_error : runtime_error
-{
-}
```

Comment:
Why ? `alias` is almost like a `#define` for type, so it would e.g. give the wrong name if used as template parameter.

Reply:
Reverted those

Reply:
and it immediately broke macos.